Rephrase to be simpler (#6)
jenkins-infra/jenkins-codeql: jenkins/java/ql/src/BadRoleCheck.md

## Why is this a problem?

This code may allow agent processes to execute code on the Jenkins controller.
As a result it could compromise security on Jenkins before 2.319 and LTS 2.303.3.

## Next Steps

<!-- Generic section used in all findings -->

1. Don't panic, even if this issue is present in previous plugin releases.

>    Do not change your code without understanding why the the finding appears.

Reviewer: Duplicate word
```suggestion
   Do not change your code without understanding why the finding appears.
```

   You may end up not fixing the problem, only hiding it instead.
2. Determine whether this finding is a false positive (see guidance below).
   This is an automated scan result, so that's always a possibility.
   In general, the rules err on the side of caution, so false positives are pretty common.
   If it is a false positive, mark it as such on the GitHub UI and you're done!

> 3. If this is a true positive finding, review the documentation below how to resolve it.

Reviewer: Alternative phrasing?
```suggestion
3. If this is a true positive finding, use the suggestions in the documentation below to resolve it.
```

Author: Or just?
```suggestion
3. If this is a true positive finding, use the documentation below to resolve it.
```
